drivers/vhm/vhm_mm.c

```diff
@@ -232,14 +231,13 @@ void deinit_trusty(struct vhm_vm *vm)
 void *map_guest_phys(unsigned long vmid, u64 guest_phys, size_t size)
 {
 	struct vhm_vm *vm;
-	void *ret = NULL;
+	void *ret;
 
 	vm = find_get_vm(vmid);
 	if (vm == NULL)
-		return ret;
+		return NULL;
 
```

Comment:
It is unnecessary to change the above code.

How about not touching the above code in patch 01? Of course if it is changed in patch 01, I think that we can continue to use it. It is not very good that it is changed back and forth.

Reply:
It's necessary.
If vm->hugetlb_enabled is enabled, the return ret is not initialized.

Reply:
@lifeix , After double checking the code, the above code needs to be changed to avoid the uninitialized variable. It is no problem.
              For the above change in patch 02, it is ok to change it back.  Of course it is no problem that it is not changed. 

Now this patch looks good to me.